mate-panel/panel-struts.c

```diff
@@ -259,6 +264,8 @@ panel_struts_set_window_hint (PanelToplevel *toplevel)
 
 	widget = GTK_WIDGET (toplevel);
 
+	g_return_if_fail(GDK_IS_X11_DISPLAY (gtk_widget_get_display (widget)));
```

Comment:
I think you love not to use a space before a bracket :)
Can you fixed that here and in other occurrences in this PR, please.

I know that we have such formatting issues in existing code........

Reply:
I'm sorry that MATE and GTK code insists on a space where it doesn't belong. I'll fix 'em

Reply:
Done